avogadro/qtplugins/qtaim/qtaimodeintegrator.cpp

```diff
@@ -796,10 +792,10 @@ qint64 QTAIMODEIntegrator::r8_rkf45(qint64 neqn, qreal y[], qreal yp[],
       } else if (kflag == 4) {
         nfe = 0;
       } else if (kflag == 5 && abserr == 0.0) {
-        exit(1);
+        // error
```

Comment:
Is is safe to do this? The code is pretty unfriendly in general, so I guess just marking it as error is OK. It would be good to rewrite some of this, but it is a decent undertaking. I don't object, as I think all of this code was pretty fragile to begin with, and was never adapted to modern C++/our general code style.

Reply:
What I'm hoping, is that the Python implementations in `cclib` and `Horton` will be able to do all of this externally, and we can jettison the C++ code in favor of 'install the Horton Python plugin scripts to run QTAIM analysis'.

Reply:
I didn't realize there were new implementations that might become available, that sounds preferable for sure! I will pay less attention to this, happy to rubber stamp it, I think this code has enough other issues I am concerned about that this shouldn't add any additional concern :-)